client/src/components/App.jsx

```diff
@@ -68,12 +75,12 @@ class App extends React.Component{
         </div>
 
         <div className="columnDivider">
-          <h3>Photo Carousel</h3>
+          {/* <h3>Photo Carousel</h3> */}
           <PhotoCarousel productName={this.state.productName} />
         </div>
 
         <div>
-          <h3>SideBar</h3>
+          {/* <h3>Side Bar</h3> */}
```

Comment:
Where these comments pushed by mistake or where they intentional?

Reply:
These are my comments. I should probably make them look more like comments, by replacing '<h3>' with 'Note'.